airbyte-integrations/connectors/source-xero/source_xero/source.py

```diff
@@ -83,8 +83,8 @@ def get_authenticator(config: Mapping[str, Any]) -> Mapping[str, Any]:
         return XeroSingleUseRefreshTokenOauth2Authenticator(
             connector_config=config,
             token_refresh_endpoint="https://identity.xero.com/connect/token",
-            client_id_config_path=["authentication", "client_id"],
-            client_secret_config_path=["authentication", "client_secret"],
+            client_id=config["authentication"]["client_id"],
+            client_secret=config["authentication"]["client_secret"],
             access_token_config_path=["authentication", "access_token"],
```

Comment:
Xero seems to be new addition. @nidhilashkari17 keep a not in case Kiwi customer we retain and have to migrate.